jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -286,6 +294,11 @@ YUI.add('deployment-flow', function() {
         // Here we need to just prevent the deployment flow to close.
         return;
       }
+      this.props.analytics.send(
+        'Deployment Flow',
+        'Button click',
+        'Close - Exit deployment'
```

Comment:
This one is tricky, do we know what it means? You can quick coz you need to do something. Or you can quit coz you don't want to deploy anymore? or you can quit coz you don't like adding creds?